MVC_Activity01/MVC_Activity01/Controllers/HomeController.cs

```diff
@@ -26,5 +27,14 @@ public ActionResult Contact()
 
             return View();
         }
+        public ActionResult OrderDetails(string orderNo, string orderDate, string customerName, string totalAmount)
+        {
+            ViewBag.Message = "View Details";
```

Comment:
You should use `Model` object instead of `ViewBag`, it is a good practice and make use of the OOP principles